playbooks/gce/openshift-cluster/launch.yml

```diff
@@ -28,33 +28,33 @@
       type: "{{ k8s_type }}"
       g_sub_host_type: "{{ sub_host_type }}"
 
-  - include: ../../common/openshift-cluster/set_node_launch_facts_tasks.yml
-    vars:
-      type: "infra"
-      count: "{{ num_infra }}"
-  - include: tasks/launch_instances.yml
-    vars:
-      instances: "{{ infra_names }}"
-      cluster: "{{ cluster_id }}"
-      type: "{{ k8s_type }}"
-      g_sub_host_type: "{{ sub_host_type }}"
-
-  - set_fact:
-      a_infra: "{{ infra_names[0] }}"
-  - add_host: name={{ a_infra }} groups=service_master
-
+#  - include: ../../common/openshift-cluster/set_infra_launch_facts_tasks.yml
+#    vars:
+#      type: "infra"
+#      count: "{{ num_infra }}"
+#  - include: tasks/launch_instances.yml
+#    vars:
+#      instances: "{{ infra_names }}"
+#      cluster: "{{ cluster_id }}"
+#      type: "{{ k8s_type }}"
+#      g_sub_host_type: "{{ sub_host_type }}"
+#
+#  - set_fact:
+#      a_infra: "{{ infra_names[0] }}"
+#  - add_host: name={{ a_infra }} groups=service_master
+#
```

Comment:
@twiest good if you can repair the infra nodes for GCE because I work without it and don't know how it works exactly